src/Arcus.WebApi.Security.Authorization/AzureManagedIdentityAuthorizationFilter.cs

```diff
@@ -31,7 +31,7 @@ public virtual async Task OnAuthorizationAsync(AuthorizationFilterContext contex
         {
             var jwtString = context.HttpContext.Request.Headers[_headerName].ToString();
 
-            if (string.IsNullOrWhiteSpace(jwtString) || !await _jwtTokenReader.IsValidToken(jwtString))
+            if (string.IsNullOrWhiteSpace(jwtString) || !await _jwtTokenReader.IsValidTokenAsync(jwtString))
```

Comment:
I'm fine with inline `await` in `if`, and inverted condition. But maybe too much for Arcus starter devs/users?

Reply:
It's not related to starter devs/users, it's just that this is less readible so I've updated this indeed.

Forgot to do that, sorry!